src/validation/validation-processor.js

```diff
@@ -21,9 +21,11 @@ const validateFile = (filePath, editorconfig) => {
 
 	fileContentArray.forEach((line, lineNumber) => {
 		lineNumber++;
-		errors.push(validateTab(line, lineNumber, editorconfig));
-		errors.push(validateSpaces(line, lineNumber, editorconfig));
-		errors.push(validateTrailingWhitespace(line, lineNumber, editorconfig));
+		if (!line.endsWith('editorconfig-disable-line')) {
```

Comment:
`endsWith` checks exactly at the strings end, which is good languages who support line comments in the form of `//` or `#`.
But in case of HTML a comment would look like `<!-- editorconfig-disable-line -->`.

I think a regex which checks for non alphanumeric chars as postfix too:
`/editorconfig-disable-line[\s-_<>]*$/` - this list may need to be extended to support all possible comment syntax.

Reply:
Thanks for this suggestion, I was nearly to writing docs and merging this, but now I will add an regex.

Reply:
I will just look for `editorconfig-disable-line` for now and create an issue to figure out a better way because it's not as easy as it might seems.